ui/components/component-library/form-text-field/form-text-field.tsx

```diff
@@ -119,6 +119,7 @@ export const FormTextField: FormTextFieldComponent = React.forwardRef(
         />
         {helpText && (
           <HelpText
+            data-testid="help-text"
```

Comment:
Should be able to use the helpTextProps to pass a data-test-id if not it's a bug so we should add it to the types

Reply:
@georgewrmarshall

Reply:
thanks @georgewrmarshall. i updated the code to use data-testid instead of fixed one on helpText. i added partial on helpTextProps so it won't require children.
https://github.com/MetaMask/metamask-extension/pull/33263/commits/f0843325900a7dfb8905572bae18333138d18390